transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__models.yml

```diff
@@ -116,3 +116,74 @@ models:
       - name: uuid_ts
       - name: sent_at
       - name: original_timestamp
+
+  - name: stg_mm_telemetry_prod__activity
+    description: Tracks key metrics related to server activity
+
+    columns:
+      - name: event_id
+        description: The event's id.
+      - name: event_name
+        description: The name of the event.
+      - name: event_table
+        description: The name of the event table.
+      - name: server_id
+        description: The ID of the server the event originated from.
+      - name: received_at
+        description: Timestamp registered by Segment when the event was ingested (received).
+      - name: timestamp
+        description: Timestamp registered by Segment when the event occured refactored to clients clock skew.
+      - name: installation_id
+        description: The id of the installation
+      - name: anonymous_id
+        description: Server's anonymous ID.
+      - name: server_ip
+        description: |
+          The server's IP address. See [Rudderstack docs](https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/#how-rudderstack-collects-ip-address) for more information.
+      - name: count_public_channels
+        description: Total number of public channels.
+      - name: count_deleted_public_channels
+        description: Total number of deleted public channels.
+      - name: count_private_channels
+        description: Total number of private channels.
+      - name: count_deleted_private_channels
+        description: Total number of deleted private channels.
+      - name: count_bot_posts_previous_day
+        description: Total number of deleted posts.
+      - name: count_posts_previous_day
+        description: Total number of posts the day before the event was sent.
+      - name: count_teams
+        description: Total number of teams in the server.
+      - name: count_slash_commands
+        description: Total number of slash commands.
+      - name: count_direct_message_channels
+        description: Total number of direct message channels.
+      - name: count_posts
+        description: Total number of posts.
+      - name: count_incoming_webhooks
+        description: Total number of incoming webhooks, excluding deleted.
+      - name: count_outgoing_webhooks
+        description: Total number of outgoing webhooks, excluding deleted.
+      - name: daily_active_users
+        description: |
+          Total number of active users for the past 24 hours prior to the timestamp of the event.
+          Bots and deleted users are excluded.
+      - name: monthly_active_users
+        description: |
+          Total number of active users for the past 31 days prior to the timestamp of the event.
```

Comment:
the `metrics` CTE in `fct_active_users.sql` is using a 30 days window - 
sum({{metric}}_last_30_days::integer) as monthly_{{column}} - not sure if we need to be aligned on 30 or 31 days across.

Reply:
Not sure as well. 31 is the max number but 30 is kind of a balancing item. 

Another idea is that if they metrics need to be super accurate, then the window should use activity event timestamp as reference. But this can become challenging in implementation.

Reply:
I think that's fine then. Maybe just something to keep in mind if we detect small inaccuracies later on.